packages/lesswrong/lib/routes.ts

```diff
@@ -914,11 +921,6 @@ const eaLwAfForumSpecificRoutes = forumSelect<Route[]>({
       getPingback: (parsedUrl) => getPostPingbackBySlug(parsedUrl, parsedUrl.params.slug),
       background: postBackground
     },
-    {
-      name: 'bookLanding',
-      path: '/books',
-      redirect: () => `/books/2018`,
-    },
```

Comment:
Seems plausible that deprecating this old redirect is what we want, but do want to double-check with @Discordius